examples/serverless.yml

```diff
@@ -15,11 +15,10 @@ functions:
       distribution: 'WebsiteDistribution'
       eventType: 'viewer-request'
       pathPattern: '/lambda'
-  onOriginRequest:
-    handler: src/handlers.onViewerRequest
-    lambdaAtEdge:
+  onOriginResponse:
+    handler: src/handlers.onViewerResponse
       distribution: 'WebsiteDistribution'
-      eventType: 'origin-request'
+      eventType: 'origin-response'
```

Comment:
Why did this need to change? There is a `onOriginResponse` just below it already.

Reply:
Not sure why I thought this needed changing. Might have mixed things up myself. Will double check why this was changed and revert if not.

Reply:
Ahh I remember now! The handler is wrong (should be onOriginRequest not onViewerRequest)